src/toil_lib/tools/aligners.py

```diff
@@ -114,6 +114,7 @@ def run_bwakit(job, config, sort=True, trim=False, mark_secondary=False):
     :rtype: str
     """
     work_dir = job.fileStore.getLocalTempDir()
+    rg = None
```

Comment:
The `if`/`elif` tree that contains assignments to `rg` doesn't have an `else`, so pycharm flags it as possibly undefined variable. We can leave this assigned to `None` or change the final conditional to `else`.